js/AppProvider.js

```diff
@@ -1,9 +1,19 @@
 import React, { Component } from 'react';
 import { Provider } from 'react-redux';
-import store from './store';
+import createStore  from './store';
 import App from './App';
 
 /**
+ * Create the global app redux store.
+ *
+ * This store is a singleton that lasts
+ * for the lifetime of the app.
+ *
+ * @type {ReduxStore}
+ */
+const store = createStore();
```

Comment:
Having `createStore()` factory allows us to create stores for unit-testing. Prior to this we only had access to a singleton.